Mu2eG4/src/constructVirtualDetectors.cc

```diff
@@ -1360,8 +1361,8 @@ namespace mu2e {
       //const double y_vd_halflength = (y_crv_max + yExtentLow)/2.0;
       const double y_mother_halflength = yExtentLow;
       const double dimVD[3] = { x_vd_halflength, y_mother_halflength, vdg->getHalfLength() };
-
-      VolumeInfo vdInfo = nestBox(VirtualDetector::volumeName(vdId),
+      /*
```

Comment:
Unless the commented out code is development work that we be uncommented when development is complete, please remove it.

Reply:
Done in [2325ea8](https://github.com/Mu2e/Offline/pull/1001/commits/2325ea879ced4042711ffb8c1bb8be320418e0f1)

Reply:
THanks